### PR TITLE
Move HTTP-related code into the Pusher\Http namespace

### PR DESCRIPTION
**Pull request by zimbatm**
Just classifying the code a bit

### CODE REVIEW

File: src/HTTP/Adapter.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace Pusher;
+namespace Pusher\HTTP;
```
Comment (vinkla) [inline src/HTTP/Adapter.php]: IMO we should use `Http` to follow the same pattern.

Comment (zimbatm) [inline src/HTTP/Adapter.php]: Fixed in ace0fcb2b34b9574442be7b3ddd41fda0bdf8a54

**Comment by zimbatm** [comment]
This is ready for merge now.

I'm getting an exception on the FileAdapterTest when running with my OSX's default PHP: `Pusher\Exception\AdapterException: file_get_contents(https://pusher.com): failed to open stream: operation failed` but it's also present in the new-lib branch. Do you guys have any ideas where that could be coming from ?

**Comment by GrahamCampbell** [comment]
You can only use https with file_get_contents if you have the openssl extension enabled i think.

**Comment by GrahamCampbell** [comment]
Also, maybe try setting this in your php.ini: `allow_url_include = On`. That would fix all http/https issues.

**Comment by zimbatm** [comment]
Thanks, opened another issue for that: https://github.com/pusher/pusher-http-php/issues/87

**Pull request merged**